Fail build testsuit in 32bits

https://github.com/KhronosGroup/Vulkan-Loader/blob/e1dc222803390e8e18e4a9a211609341ea34ccab/tests/loader_regression_tests.cpp#L474-L480

~~~
[ 95%] Built target vk_loader_validation_tests
/tmp/makepkg/sl1-lib32-vulkan-git/src/Vulkan-Loader/tests/loader_regression_tests.cpp:477:6: error: #elif with no expression
  477 | #elif
      |      ^
make[2]: *** [tests/CMakeFiles/test_regression.dir/build.make:90: tests/CMakeFiles/test_regression.dir/loader_regression_tests.cpp.o] Error 1
~~~

mybe need set 

~~~
#elif defined(__i386__)
~~~

?

related https://github.com/KhronosGroup/Vulkan-Loader/commit/d7c8139c370a601e9637c21eeedb95d98fe17703

greetings

--- a/tests/loader_regression_tests.cpp
+++ b/tests/loader_regression_tests.cpp
@@ -474,7 +474,7 @@ TEST(TryLoadWrongBinaries, WrongICD) {
 #if defined(__linux__)
 #if defined(__x86_64__)
     ASSERT_TRUE(log.find("wrong ELF class: ELFCLASS32"));
-#elif
+#else
     ASSERT_TRUE(log.find("wrong ELF class: ELFCLASS64"));
 #endif
 #endif

